OrbitGl/Track.cpp

```diff
@@ -87,7 +87,7 @@ void Track::Draw(GlCanvas* canvas, PickingMode picking_mode, float z_offset) {
   float y0 = pos_[1];
   float y1 = y0 - size_[1];
   float track_z = GlCanvas::kZValueTrack + z_offset;
-  float text_z = GlCanvas::kZValueText + z_offset;
+  float text_z = GlCanvas::kZValueTrack + z_offset;
```

Comment:
Because kZValueText is on the overlay layer, but I need the text to be on the same layer as the track itself, otherwise the text of underlying tracks will be on top of the moving track.

I'm trying to minimize the usages of different z-values. IMO, text should be on the same layer as the element on which it is drawn, so it should be on the same level as the track background, because it is part of the track. Since it's drawn after the track background, it will correctly show on top of the track.

Reply:
You are right!!